ql/src/test/results/clientpositive/llap/multi_count_distinct_null.q.out

```diff
@@ -73,7 +73,7 @@ STAGE PLANS:
                 outputColumnNames: _col0, _col1, _col2, _col3
                 Statistics: Num rows: 18 Data size: 1628 Basic stats: COMPLETE Column stats: COMPLETE
                 Select Operator
-                  expressions: CASE WHEN (((_col3 = 3L) and _col0 is not null)) THEN (1) ELSE (null) END (type: int), CASE WHEN (((_col3 = 5L) and _col1 is not null)) THEN (1) ELSE (null) END (type: int), CASE WHEN (((_col3 = 6L) and _col2 is not null)) THEN (1) ELSE (null) END (type: int)
+                  expressions: CASE WHEN (((_col3 = 3L) and _col0 is not null)) THEN (1) ELSE (UDFToInteger(null)) END (type: int), CASE WHEN (((_col3 = 5L) and _col1 is not null)) THEN (1) ELSE (UDFToInteger(null)) END (type: int), CASE WHEN (((_col3 = 6L) and _col2 is not null)) THEN (1) ELSE (UDFToInteger(null)) END (type: int)
```

Comment:
Cast not being constant folded?

Reply:
Latest update fixes constant folding issue.